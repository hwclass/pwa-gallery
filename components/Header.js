export default class extends React.Component {
  render() {
    return (
      <header>Header</header>
    )
  }
}