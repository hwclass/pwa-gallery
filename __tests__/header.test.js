/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../components/Header.js'

describe('With Enzyme', () => {
  it('Header shows "Header"', () => {
    const header = shallow(<Header />)

    expect(header.text()).toEqual('Header')
  })
})

describe('With Snapshot Testing', () => {
  it('Header shows "Header"', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})