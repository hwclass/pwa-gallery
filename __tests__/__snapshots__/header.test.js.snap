// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Header shows "Header" 1`] = `
<header>
  Header
</header>
`;
